terraform/20-app/lambda.api-gateway.tf

```diff
@@ -0,0 +1,43 @@
+data "aws_caller_identity" "current" {}
+
+module "api_gateway_lambda" {
+  source        = "terraform-aws-modules/lambda/aws"
+  version       = "7.8.1"
+  function_name = "${local.prefix}-api-gateway-auth-lambda"
+  description   = "Handles API Gateway authentication requests."
+
+  runtime       = "nodejs18.x"
+  architectures = ["arm64"]
+  handler       = "index.handler"
+
+  create_package = true
+  package_type   = "Zip"
+  source_path    = "../../src/lambda-api-gateway"
+
+  timeout        = 15
+  memory_size    = 128
+
+  cloudwatch_logs_retention_in_days = 7
+
+  environment_variables = {
+    SECRET_NAME         = "${local.prefix}-cognito-service-credentials"
+    TENANT_SECRET_NAME  = "${local.prefix}-ukhsa-tenant-id"
```

Comment:
Are we still saving this secret in secrets manager on a per environment basis? You have access to this at `var.ukhsa_tenant_id` in terraform, so you'd need to pass that in here instead

Reply:
Good catch, removed